tests/probe/t15-specimen.sh

```
@@ -0,0 +1,66 @@
+#
+#
+#
+#
+#
+#
+# ---------------------------------------------------------------- region B ---
+
+write_report() {
+  OUT=$1
+  shift
+  rm -f $OUT
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)